gspread/spreadsheet.py

```diff
@@ -739,3 +739,33 @@ def list_protected_ranges(self, sheetid):
             raise WorksheetNotFound("worksheet id {} not found".format(sheetid))
 
         return sheet.get("protectedRanges", [])
+
+    def get_all_worksheet_values(self, skip_worksheet_titles: list = None):
+        """Grabs all the data from all the worksheets in one API call. Skips any worksheets that were named in the
+        skip_worksheet_title param.
+
+        :param list skip_worksheet_titles: A list of worksheet titles to skip.
+        :returns: a dict of worksheet data with worksheet title as key
+        :rtype: dict
+        """
+
+        if skip_worksheet_titles is None:
+            skip_worksheet_titles = []
+
```

Comment:
if you set it to `[]` empty list when it's not set, then just set the default value to `[]` in the argument in the method definition.

Reply:
I suppose the initial solution was right, because usage of a mutable object as default argument can follow wrong behaviour. You can see example of such problem [there](https://realpython.com/python-mutable-vs-immutable-types/#using-mutable-default-values)